tooling/template-base-essentials/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@thoughtindustries/helium-template-essentials",
+  "version": "1.2.0-beta.6",
+  "scripts": {
+    "authenticate": "helium authenticate",
+    "deploy": "helium deploy",
+    "dev": "helium dev",
+    "update-translations": "helium update-translations",
+    "dev:vite": "concurrently \"npm run dev:css\" \"npm run dev:server\"",
+    "dev:css": "tailwindcss -o ./renderer/tailwind.css --watch",
+    "dev:server": "tsup ./server/index.ts && node ./dist/index.js",
+    "build": "npm run build:css && npm run build:vite",
+    "build:css": "tailwindcss -o ./renderer/tailwind.css",
+    "build:vite": "vite build",
+    "build:atoms": "vite build -c vite.atoms.config.ts && vite build"
```

Comment:
Can remove this command since `atoms` isn't included -- same in `template-base`.